kafka-ui-api/src/main/java/com/provectus/kafka/ui/controller/KafkaConnectController.java

```diff
@@ -161,95 +178,100 @@ public Mono<ResponseEntity<ConnectorDTO>> setConnectorConfig(String clusterName,
                                                                Mono<Map<String, Object>> requestBody,
                                                                ServerWebExchange exchange) {
 
-    Mono<Void> validateAccess = accessControlService.validateAccess(AccessContext.builder()
+    var context = AccessContext.builder()
         .cluster(clusterName)
         .connect(connectName)
         .connectActions(ConnectAction.VIEW, ConnectAction.EDIT)
-        .build());
+        .auditOperation("setConnectorConfig")
```

Comment:
let's log connector name in operation context?

Reply:
done